src/ttkbootstrap/widgets.py

```diff
@@ -95,7 +96,7 @@ class DateEntry(ttk.Frame):
     def __init__(
         self,
         master=None,
-        dateformat=r"%Y-%m-%d",
+        dateformat=locale.nl_langinfo(locale.D_FMT),
```

Comment:
I like this idea, but unfortunately, this method is [not available on Windows](https://www.w3schools.com/pHP//func_string_nl_langinfo.asp#:~:text=The%20nl_langinfo%20%28%29%20function%20returns%20specific%20local%20information.,information%2C%20the%20nl_langinfo%20%28%29%20function%20returns%20specific%20information.). We need something that is cross-platform compatible.